src/systems/tracked-controls.js

```diff
@@ -16,15 +18,20 @@ module.exports.System = registerSystem('tracked-controls', {
     });
   },
 
-  tick: function () {
-    var gamepads = navigator.getGamepads && navigator.getGamepads();
-    var gamepad;
+  rebuildControllerList: function () {
     var controllers = this.controllers = [];
-    var i;
-    if (!gamepads) { return; }
-    for (i = 0; i < gamepads.length; ++i) {
-      gamepad = gamepads[i];
+    var gamepads = trackedControlsUtils.getGamepadsByPrefix('');
+    for (var i = 0; i < gamepads.length; i++) {
+      var gamepad = gamepads[i];
       if (gamepad && gamepad.pose) { controllers.push(gamepad); }
     }
+  },
+
+  tick: function (now) {
+    if (now < this.lastControllerCheck + 10) { return; }
+
+    this.lastControllerCheck = now;
+    this.rebuildControllerList();
+    this.sceneEl.emit('tracked-controls.tick', { timestamp: now, controllers: this.controllers });
```

Comment:
I would call the event `gamepadsupdated`. Is it useful to pass the time? Is it used anywhere?